frontend/packages/shared/src/publication-content.tsx

```diff
@@ -399,8 +403,12 @@ export function BlockNodeContent({
           index={props.index}
           start={props.start}
         />
-        <BlockContent block={blockNode.block!} depth={depth} {...hoverProps} />
-        {!props.embedDepth ? (
+        <BlockContent
+          block={blockNode.block!}
+          depth={depth}
+          {...interactiveProps}
+        />
+        {!props.embedDepth && !renderOnly ? (
```

Comment:
why should we have this renderOnly? We can just detect when all the features are off, like if onCopyBlock and onReplyBlock are null, and if there are no citations. I don't think we need to add another prop

Reply:
this is to prevent firing the onMouseOver events. specially those